src/main/java/com/bzabek/ticketbookingapp/repository/RoomSpringDataRepository.java

```diff
@@ -0,0 +1,13 @@
+package com.bzabek.ticketbookingapp.repository;
+
+import com.bzabek.ticketbookingapp.entities.Room;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Repository;
+
+import java.util.Optional;
+
+@Repository
+public interface RoomSpringDataRepository extends JpaRepository<Room,Long> {
+    @Override
+    Optional<Room> findById(Long aLong);
```

Comment:
mogę przekazać null jako parametr?

Reply:
Możesz lecz nie zwróci to żadnego rekordu. Jeśli nie wykorzystamy metod klasy Optional (np. orElse, orElseThrow) doprowadzi to do błędu.